frontend/src/components/layout/SearchingOverlay.vue

```diff
@@ -1,8 +1,14 @@
 <template>
    <div v-show="searching" class="searching-overlay">
       <div class="searching-box">
-         <p>{{message}}</p>
-         <img src="../../assets/searching.gif">
+        <div class="book">
+          <div class="book-page"></div>
```

Comment:
Why the book div and 3 empty book-page divs?

Reply:
nevermind... i see why later on in the changes

Reply:
Argh, I forgot to clean out that commented code. 
Those empty divs are the "pages" of the book animation.

Reply:
I don't have a current way to test this in Windows (I did on my old laptop - hoping to get VMware installed soon)
If this doesn't work in Windows we can roll back to using an animated gif.